class_defines.py

```diff
@@ -2383,6 +2433,14 @@ def init_props(self, **kwargs):
         e1, e2 = self.entity1, self.entity2
         if e1.is_line():
             value = e1.length
+        elif type(e1) in curve:
+            centerpoint = e1.ct.co
+            if isinstance(e2, SlvsLine2D):
+                endpoint, whatever = intersect_point_line(centerpoint, e2.p1.co, e2.p2.co)
```

Comment:
Usually "_" is used as a name for variables that aren't actually used, "whatever" as a name might look like it's actually used somewhere